lib/simctl.js

```diff
@@ -364,8 +366,13 @@ async function createDevice (name, deviceTypeId, runtimeId, opts = {}) {
     log.warn(`Unable to find runtime for iOS '${runtimeId}'. Continuing`);
   }
 
-  const cltVersion = await getCommandLineToolsVersion();
-  if (util.compareVersions(cltVersion, '>=', CMDLINE_TOOLS_FORMAT_CHANGED_VERSION)) {
+  if (util.compareVersions(await getCommandLineToolsVersion(), '>=', CMDLINE_TOOLS_FORMAT_CHANGED_VERSION)) {
```

Comment:
can this be combined into util.compareversion(await getCommandLineToolsVersion() ...) || util.compareversion(getVersion(false) ...) then ... ?

Reply:
yes, I already call getting versions in compareVersions